src/core/server/saved_objects/service/lib/repository.ts

```diff
@@ -1287,21 +1656,24 @@ export class SavedObjectsRepository {
     id: string,
```

Comment:
Was this function intentionally excluded from `repository.security_extension.test.ts`?

Reply:
Nevermind, I found these tests elsewhere!